.github/workflows/build-test.yml

```diff
@@ -38,12 +38,12 @@ jobs:
                 shell: cmd
                 working-directory: ./ci-tools/windows
 
-          - os: ubuntu-16.04
+          - os: ubuntu-20.04
             sys_pkgs: sudo apt install libboost-dev libboost-all-dev
-            build_unit_test: make.sh -t -g "Unix Makefiles"
-            build_reg_test:  make.sh -g "Unix Makefiles"
+            build_unit_test: make.sh -s -t -g "Unix Makefiles"
```

Comment:
hey @karosc these edits look great! sorry to be pedantic, can you add a comment on why we use these flags? might be useful in the next round of updates.

Reply:
added a comment